massa-api/src/public.rs

```diff
@@ -361,30 +357,24 @@ impl Endpoints for API<Public> {
             let operation_ids: Set<OperationId> = ops.iter().cloned().collect();
 
             // simultaneously ask pool and consensus
-            let (pool_res, consensus_res): (
-                Result<(Map<OperationId, OperationSearchResult>, Storage), ConsensusError>,
-                Result<(Map<OperationId, OperationSearchResult>, Storage), ConsensusError>,
-            ) = tokio::join!(
-                //TOxDO: https://github.com/massalabs/massa/issues/2866
-                //pool_command_sender.get_operations(operation_ids.clone()),
-                async { Ok((Map::default(), Storage::default())) },
-                consensus_command_sender.get_operations(operation_ids)
-            );
-            let (pool_res, consensus_res) = (pool_res?, consensus_res?);
+            let consensus_res = consensus_command_sender
+                .get_operations(operation_ids)
+                .await?;
+            let pool_res = pool_command_sender.get_operations();
```

Comment:
The get operations is made with specific ids. Then, there is no other usage of `get_operations()` and so I think we shouldn't create this function because it's very heavy to run and don't think there is use case for this.

Reply:
I'll take by id :+1: but I need the operations bellow. I don't get your second point.